python/ray/dag/compiled_dag_node.py

```diff
@@ -379,6 +381,20 @@ class CompiledDAG:
     information.
     """
 
+    @ray.remote
```

Comment:
```suggestion
    @ray.remote(num_cpus=0)
```

The default is 0 but let's be explicit to be safe.